apps/marginfi-v2-trading/src/config/trade.ts

```diff
@@ -2,5 +2,5 @@ export const TRADE_GROUPS_MAP = "https://storage.googleapis.com/mrgn-public/mfi-
 export const TOKEN_METADATA_MAP =
   "https://storage.googleapis.com/mrgn-public/mfi-trade-token-metadata-cache.json?invalidatecache=1103";
 export const BANK_METADATA_MAP =
-  "https://storage.googleapis.com/mrgn-public/mfi-trade-bank-metadata-cache.json?invalidatecache=1103";
+  "https://storage.googleapis.com/mrgn-public/mfi-trade-bank-metadata-cache-test.json?invalidatecache=1103";
```

Comment:
@k0beLeenders I added groupAddress to bank metadata. did not want to disrupt prod so will be switching this during merge.